### PR TITLE
Fix CVE on NodeJS by adding install line (WIP)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR is related to meshery/meshery#4642

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,6 +29,8 @@ ENV DISTRO="debian"
 ENV GOARCH="amd64"
 ENV SERVICE_ADDR="meshery-nginx-sm"
 ENV MESHERY_SERVER="http://meshery:9081"
+RUN npm i ansi-regex@latest
```
Comment (Aisuko) [inline Dockerfile:32]: Is there we can just add one line of command to reduce extra container layers?

Comment (alphaX86) [reply Dockerfile:32]: Problem is that I couldn't interact with the distro less image. Is there any way to install a npm package?

Comment (alphaX86) [reply Dockerfile:32]: The line is typed to fetch the latest version of the ansi-regex package

**Comment by alphaX86** [comment]
Making PR as draft as direct usage of commands is not possible in GCr distroless

File: Dockerfile (hunk 1 above)
Comment (Aisuko) [reply Dockerfile:32]: The distro less image is target to no interact of the application image. So, there may not easy to install npm packages I guess.

Comment (Aisuko) [reply Dockerfile:32]: Any help of this ? https://github.com/GoogleContainerTools/distroless/blob/main/examples/nodejs/Dockerfile

Comment (alphaX86) [reply Dockerfile:32]: Will check it out 👍

**Pull request closed without merging**